### PR TITLE
Add toggle for user profile to realm settings

### PR DESCRIPTION
**Pull request by jonkoops**
## Verification Steps
1. Start the Keycloak server with the `DECLARATIVE_USER_PROFILE` feature enabled (`start.mjs` will do this for you).
2. Go to realm settings and enable the 'User Profile Enabled' switch
3. Verify the 'User Profile' tab is visible

### CODE REVIEW

File: src/realm-settings/GeneralTab.tsx [hunk 1]
```diff
@@ -166,6 +168,35 @@ export const RealmSettingsGeneralTab = ({
             )}
           />
         </FormGroup>
+        {isFeatureEnabled(Feature.DeclarativeUserProfile) && (
```
Comment (edewit) [inline src/realm-settings/GeneralTab.tsx:171]: shouldn't it be if feature enabled and user profile realm switch enabled?

**Review by edewit — APPROVED** [review]

**Review by mfrances17 — APPROVED** [review]
LGTM, works as expected, nice job. The approval is assuming that the `Attributes` and `Attributes group` tab contents are incomplete and TBD.

File: src/realm-settings/GeneralTab.tsx (hunk 1 above)
Comment (jonkoops) [reply src/realm-settings/GeneralTab.tsx:171]: Yeah, this condition is for the toggle itself not the tab (see [here](https://github.com/keycloak/keycloak-admin-ui/pull/1717/files#diff-385f6ae54f2531fbacfd9d3b9036588968189956cb45d8ccb473f566e58b5707R434-R435))

**Comment by jonkoops** [comment]
> The approval is assuming that the `Attributes` and `Attributes group` tab contents are incomplete and TBD.

Yes, that is correct. The rest of the implementation will follow in a couple of smaller PRs to keep the review workload to a minimum.

**Pull request merged**